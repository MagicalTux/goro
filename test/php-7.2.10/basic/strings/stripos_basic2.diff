--
     bool(false)
     int(7)
     int(7)
007- int(7)
008- bool(false)
009- int(7)
007+ int(1)
008+ int(4)
009+ int(-5)
     int(0)
     int(0)
     bool(false)
     int(0)
     bool(false)
015- int(0)
016- bool(false)
015+ int(-12)
016+ int(-11)
     int(4)
     int(8)
     int(8)
     bool(false)
021- int(8)
022- int(4)
023- int(4)
024- int(8)
025- bool(false)
021+ int(-3)
022+ int(-4)
023+ int(-6)
024+ int(0)
025+ int(1)
     *** Done ***
