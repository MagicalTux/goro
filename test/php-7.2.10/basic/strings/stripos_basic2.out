*** Testing stripos() function: basic functionality ***
-- With all arguments --
int(0)
bool(false)
int(7)
int(7)
int(1)
int(4)
int(-5)
int(0)
int(0)
bool(false)
int(0)
bool(false)
int(-12)
int(-11)
int(4)
int(8)
int(8)
bool(false)
int(-3)
int(-4)
int(-6)
int(0)
int(1)
*** Done ***